phpcs.xml.dist

```diff
@@ -5,4 +5,9 @@
     <rule ref="CakePHP" />
 
     <exclude-pattern>*/comparisons/*</exclude-pattern>
+
+    <rule ref="CakePHP">
+        <!-- Exclude unwanted sniffs -->
+        <exclude name="Generic.Commenting.Todo.TaskFound" /> <!-- Excluded because it triggers on Todo* models -->
```

Comment:
This shouldn't be required, the Todo sniff has already been removed from codesniffer's `next` branch.

Reply:
I don't think bake is using that version yet.

Reply:
Interesting, I was trying to work around the stickler errors. Stickler doesn't have the dev-next branch installed.